### ISSUE TITLE
Build frontend without user interaction

### ISSUE THREAD

**Issue by jjokella** [HPSCTerrSys/TSMP2#45]
Currently the build frontend requires user interaction - in particular, when submodules are already present.

Maybe we can discuss refactoring this in such a way that automatic builds are more easily possible. Especially helpful for automatic building.

Current workaround `yes n | ./build_tsmp2.sh ...`

**Comment by kvrigor** [comment]
This will be addressed in #53 via the `--no_update` option.

**Comment by s-poll** [comment]
It would be convenient to have an option `--take_defaults`for the buildbot-system.

**Comment by jjokella** [comment]
I agree that the number of options for `build_tsmp2.sh` should be kept minimal and that the code of `build_tsmp2.sh` should be kept as simple as possible.

I would maintain that the user interaction (having to type "y") should be removed.

Striving for a minimal `build_tsmp2.sh`, maybe it would be an option to simply always use:
```bash
git submodule update --init --force -- ${submodule_name}
```

In a sense this suggestion goes into the opposite direction of my original request. But it complies with "Keep it simple" and it brings clarity in the sense that the exact submodule commits are used, except the `*_SRC` options are used.
